CoEpi/domain/model/UnixTime.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+struct UnixTime: Codable {
+    let value: Int64
+
+    static func minTimestamp() -> UnixTime {
+        UnixTime(value: 0)
+    }
+
+    static func now() -> UnixTime {
+        UnixTime(value: Int64(Date().timeIntervalSince1970))
```

Comment:
@i-schuetz This might be a good point to switch to millisecond resolution that V4 backend is expecting.

Reply:
There has not been a final decision on the backend resolution yet.